### DISCUSSION TITLE
Does ESP32S work?

### DISCUSSION THREAD

**Discussion by sudo-lupus666** [darthcloud/BlueRetro#66] [Q&A] [answered]
Hello, 

First of all: Thank you so much for sharing your project with the community. We are so lucky for having people so talented as you are.

Now, My question: I've got 2 ESP32 - WROOM - 32D units. I've flashed the firmware and wired one for PS - PS2, but I got nothing but a red steady red light. I've made the pins conversion because your project uses an ESP32 V4 and I don't know what version is mine, but pins are in different locations. Also, manual says my boards are powered by 3.3V and I know PS2 provides a 3.something wire. 
- Can I just plug it into the 3.3V pin? 
- Does my board work?

**Comment by sudo-lupus666** [comment, marked as answer]
After some tries, I got it working! 

- Pins conversion works ok on ESP32 - WROOM - 32D - version something.
- PS2 provides 3.3 and 8v. I used 8v in vin pin.

I'm so happy. I just tried my Dual Sense on PS1 and it works great! Thank you so much! I'll upload some pictures with my (ugly) work.

**Reply by sudo-lupus666** [reply]
- [ ] ![image](https://user-images.githubusercontent.com/83513847/131240169-489cbcfa-e37d-4b96-aa90-1cd3805e0262.jpeg)![image](https://user-images.githubusercontent.com/83513847/131240191-6a364b1a-3325-46c9-ac5d-8b2e72fd0c9f.jpeg)![image](https://user-images.githubusercontent.com/83513847/131240203-c095461c-63df-4c49-83e9-aa51279f3fc9.jpeg)

**Reply by sudo-lupus666** [reply]
This is kind of a disaster, aesthetically speaking, but it works! Btw I didn't connect second player's pin 3, but I did connect ground. Any problems? 

One more time: thank you. <3

**Reply by darthcloud** [reply]
It's fine to connect only P1 8V. Glad you like it!
